src/pages/PassportRedemption/Redemption/RedemptionFooters.tsx

```diff
@@ -45,7 +46,7 @@ export const RedeemRewardsFooter = (props: RedeemRewardsProps) => {
           window.location.href = `/passport/${props.id}/redeem/${props.access_token}/sponsor/${props.selectedSponsor.id}`;
         }}
       >
-        {t('passport.placeholders.redeemNow').toUpperCase()}
+        REDEEM NOW
```

Comment:
Why did we change to hardcoding? Same with the other text in the file.

Reply:
ah theres some files that have hardcoded translations because i missed them when i had to do the rebase of dev with the i18n. Will fix then merge